test/ctia/bundle/routes_test.clj

```diff
@@ -908,17 +915,17 @@
            (is (= bundle-incident-target-get bundle-incident-target-post))))))))
 
 (defn with-tlp-property-setting [tlp f]
-  (with-redefs [p/global-properties-atom
-                (let [new-props (-> (p/get-global-properties)
-                                    (assoc-in [:ctia :access-control :min-tlp] tlp)
-                                    (assoc-in [:ctia :access-control :default-tlp] tlp)
-                                    atom)]
-                  (fn [] new-props))]
-    (f)))
+  (helpers/with-config-transformer*
+    #(-> %
+         (assoc-in [:ctia :access-control :min-tlp] tlp)
+         (assoc-in [:ctia :access-control :default-tlp] tlp))
+    f))
 
 (deftest bundle-tlp-test
-  (test-for-each-store
-   (fn []
+ (with-tlp-property-setting "amber"
```

Comment:
We must bring `with-tlp-property-setting` outside `test-for-each-store` so the former can change the config via `with-config-transformer*`  before the latter starts an app.